parts_list/master_parts_list_raw.csv

```diff
@@ -2,8 +2,8 @@ Part Name,Project Ref Code,Model/Config,Site,Link,Sold in Packs of,Total Used in
 #6-32 x 1/4 Button Head Screw,B1,92949A144,McMaster,https://www.mcmaster.com/92949a144,100,228,3,$3.45 ,$10.35,"Body, Corner Steering, Differential Pivot, Mechanical Integration, Wheel Assembly"
 #6-32 x 3/8 Button Head Screw,B2,92949A146,McMaster,https://www.mcmaster.com/92949A146,100,72,1,$3.72 ,$3.72,"Body, Head Assembly, PCB Assembly, Rocker-Bogie"
 #6-32 x 1/2 Button Head Screw,B3,92949A148,McMaster,https://www.mcmaster.com/92949A148,100,8,1,$3.72 ,$3.72,Wheel Assembly
-#6-32 x 5/8 Button Head Screw,B4,92949A150,McMaster,https://www.mcmaster.com/92949A150,100,32,1,$4.07 ,$4.07,Rocker-Bogie
-#6-32 x 3/4 Button Head Screw,B5,92949A151,McMaster,https://www.mcmaster.com/92949A151,100,28,1,$4.41 ,$4.41,"Corner Steering, Wheel Assembly"
+#6-32 x 5/8 Button Head Screw,B4,92949A150,McMaster,https://www.mcmaster.com/92949A150,100,44,1,$4.07 ,$4.07,Rocker-Bogie
+#6-32 x 3/4 Button Head Screw,B5,92949A151,McMaster,https://www.mcmaster.com/92949A151,100,28,1,$4.41 ,$4.41,Corner Steering, Wheel Assembly

```

Comment:
```suggestion
#6-32 x 3/4 Button Head Screw,B5,92949A151,McMaster,https://www.mcmaster.com/92949A151,100,28,1,$4.41 ,$4.41, "Corner Steering, Wheel Assembly"

```

this doesn't render correctly otherwise

I suggest always looking at the rendered version of these files before pushing your commits